pymzml/run.py

```diff
@@ -220,14 +229,7 @@ def _determine_file_encoding(self, path):
         mzml_encoding = 'utf-8'
```

Comment:
Please change to:

```
if os.path.exists(path):
    if path.endswith('.gz') or path.endswith('.igz'):
        import gzip
        _open = gzip.open
    else:
        _open = open
    with open(path, 'rb') as sniffer:
        header = sniffer.readline()
```
So we can also guess the encoding in gzip compressed files